pkg/logging/config.go

```diff
@@ -59,3 +62,22 @@ func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*logging.Config, error
 func UpdateLevelFromConfigMap(logger *zap.SugaredLogger, atomicLevel zap.AtomicLevel, levelKey string) func(configMap *corev1.ConfigMap) {
 	return logging.UpdateLevelFromConfigMap(logger, atomicLevel, levelKey, components...)
 }
+
+func ConfigName() string {
```

Comment:
Same comment here as in the eventing pr:
https://github.com/knative/eventing/pull/842#discussion_r261959541

Not sure I understand the solution about the superset being the solution, but I'd prefer not to have a configmap that has a pointer to another configmap.

Reply:
The code update follow by the `eventing`

Reply:
ok, so the plan is:
 1. make the name of the configmap configurable (as in the simpler PR in eventing)
 2. various configmaps will get merged somehow

Or the user can manually create one merged logging file by specifying the same file in step 1 above?

Reply:
User can manually create one merged logging file include the following content
```
    loglevel.controller: "info"
    loglevel.autoscaler: "info"
    loglevel.queueproxy: "info"
    loglevel.webhook: "info"
    loglevel.activator: "info"
    loglevel.creds-init: "info"
    loglevel.git-init: "info"
```